### PR TITLE
include ExcludeInboundPorts and ExcludeOutboundPorts parameters

### PR DESCRIPTION
**Pull request by hcavarsan**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | 
| License         | Apache 2.0

### What's in this PR?

ExcludeOutboundPorts and  ExcludeInboundPorts is configurable.

### Why?

add new parameters in Istio CRD

### Additional context

necessary to configure the ports where not to capture egress traffic

### Checklist

- [x] Implementation tested
- [x] Error handling code meets the [guideline](https://github.com/banzaicloud/pipeline/blob/master/docs/error-handling-guide.md)
- [x] Logging code meets the guideline
- [x] User guide and development docs updated (if needed)

### CODE REVIEW

File: pkg/apis/istio/v1beta1/istio_types.go [hunk 1]
```diff
@@ -790,6 +790,12 @@ type IstioSpec struct {
 	// ExcludeIPRanges the range where not to capture egress traffic
 	ExcludeIPRanges string `json:"excludeIPRanges,omitempty"`
 
+	// ExcludeInboundPorts the ports where not to capture egress traffic
```
Comment (Laci21) [inline pkg/apis/istio/v1beta1/istio_types.go]:
```suggestion
	// ExcludeInboundPorts the comma separated list of inbound ports to be excluded from redirection to Envoy
```

File: pkg/apis/istio/v1beta1/istio_types.go [hunk 2]
```diff
@@ -790,6 +790,12 @@ type IstioSpec struct {
...
+	ExcludeInboundPorts string `json:"excludeInboundPorts,omitempty"`
+
+    // ExcludeOutboundPorts the ports where not to capture egress traffic
```
Comment (Laci21) [inline pkg/apis/istio/v1beta1/istio_types.go]:
```suggestion
  // ExcludeOutboundPorts the comma separated list of outbound ports to be excluded from redirection to Envoy.
```

File: pkg/apis/istio/v1beta1/remoteistio_types.go [hunk 3]
```diff
@@ -56,6 +56,12 @@ type RemoteIstioSpec struct {
 	// ExcludeIPRanges the range where not to capture egress traffic
 	ExcludeIPRanges string `json:"excludeIPRanges,omitempty"`
 
+	// ExcludeInboundPorts the ports where not to capture egress traffic
```
Comment (Laci21) [inline pkg/apis/istio/v1beta1/remoteistio_types.go]:
```suggestion
	// ExcludeInboundPorts the comma separated list of inbound ports to be excluded from redirection to Envoy
```

File: pkg/apis/istio/v1beta1/remoteistio_types.go [hunk 4]
```diff
@@ -56,6 +56,12 @@ type RemoteIstioSpec struct {
...
+	ExcludeInboundPorts string `json:"excludeInboundPorts,omitempty"`
+	
+	// ExcludeOutboundPorts the ports where not to capture egress traffic
```
Comment (Laci21) [inline pkg/apis/istio/v1beta1/remoteistio_types.go]:
```suggestion
	// ExcludeOutboundPorts the comma separated list of outbound ports to be excluded from redirection to Envoy.
```

**Review by Laci21 — COMMENTED** [review]
Thank you for your PR, @hcavarsan!

I think it's all good, I just have one suggestion regarding the description of the fields as you can see below. (I got the wording from here: https://preliminary.istio.io/latest/docs/reference/config/annotations/) If you agree with that change, pls regenerate the CRDs as well.

**Review by waynz0r — APPROVED** [review]

**Comment by hcavarsan** [comment]
> Thank you for your PR, @hcavarsan!
> 
> I think it's all good, I just have one suggestion regarding the description of the fields as you can see below. (I got the wording from here: https://preliminary.istio.io/latest/docs/reference/config/annotations/) If you agree with that change, pls regenerate the CRDs as well.

done @Laci21, thanks!

**Review by Laci21 — APPROVED** [review]
lgtm, thank you for the contribution!

**Comment by Laci21** [comment]
@hcavarsan, fyi, this is released in image version `0.9.6`, chart version `0.0.84`.

**Pull request merged**